tutorcruncher/hermes
# Fix termination category

Close https://github.com/tutorcruncher/TutorCruncher2/issues/13663

## Description

before posting to tc2 clients we modify the `extra_attrs` `termination_category_value` so that it replaces the space with  -

we do this because TC2 expects dropdown values to be lowercase  stripped and with a dash between words


### Recreation steps:
 - follow new readme to setup pipedrive hermes and TC2
 - look at docs  / live meta tc2 to see how the Termination Category custom field is setup
 - setup Termination category on meta
 - sign up company
 - check company is create in hermes and pd
 - mofiy meta client Termination category to one of drop down
 - API error in termination category formatting
 
### Testing steps:
- do above
- termination category shouldn't cause any issues
- once you have set the termination category the first time, you can just modify any other field like utm category in TC2 or Pipedrive and see that the webhook back to TC2 from hermes does not cause an error
